apis/apim/v1alpha1/ratelimitpolicy_types.go

```diff
@@ -103,24 +103,29 @@ type RateLimit struct {
 	Actions []*ActionSpecifier `json:"actions,omitempty"`
 }
 
+// Each operation type has OR semantics and overall AND semantics for a match.
 type Operation struct {
 	Paths   []string `json:"paths,omitempty"`
 	Methods []string `json:"methods,omitempty"`
 	Hosts   []string `json:"hosts,omitempty"`
 }
 
 type Rule struct {
+	// Name supports regex for fetching operations from routing resources
+	// For VirtualService, if route name matches, all the match requests are
+	// converted to operations internally. But specific match request names
+	// are also supported.
 	Name string `json:"name,omitempty"`
 	// Operation specifies the operations of a request
 	// +optional
-	Operation *Operation `json:"operation,omitempty"`
+	Operation []*Operation `json:"operations,omitempty"`
```

Comment:
nit: strutct field should also be named `Operations`?

Reply:
Done